### PR TITLE
implement IosCanvas.drawCircle()

### PR DESCRIPTION
**Pull request by Longri**

### CODE REVIEW

File: vtm-ios/build.gradle [hunk 1]
```diff
@@ -45,6 +45,11 @@ task copyVtmThemesResources(type: Copy) {
     into("${buildDir}")
 }
 
+task copyVtmPlaygroundResources(type: Copy) {
+    from("../vtm-playground/resources")
+    into("${buildDir}/assets/")
+}
+
```
Comment (devemux86) [inline vtm-ios/build.gradle]: Do we want that?
`vtm-ios` is a published library, while `vtm-playground` is a sample app.

**Comment by devemux86** [comment]
Probably a `vtm-ios-example` module is needed to host iOS samples?

**Comment by Longri** [comment]
You are right, I will delete my iOS sample in the next 10 minutes

**Comment by Longri** [comment]
Should I create an iOS-playground?

**Comment by devemux86** [comment]
Wouldn't be better `vtm-ios-example` to be side by side with other mobile `vtm-android-example`?

**Comment by devemux86** [comment]
Could you include the `vtm-ios-example` in a separate PR and keep this for the `drawCircle` only?

**Comment by Longri** [comment]
Done!
Sorry, it was 26min instead of 10min

**Comment by devemux86** [comment]
(I can merge them separately, but it's better for the repository history)

**Comment by Longri** [comment]
on Master?
the Master has no ClusterLayer, so I must wait

**Comment by devemux86** [comment]
If the new module contains cluster example, then please wait for the cluster merge first.

I mean more here, to clean the PR leaving only the `drawCircle` implementation.

**Comment by devemux86** [comment]
Thanks!
Merged the `drawCircle` part via 325f848969575d09ebd27e4fff76da56112a8367.

**Comment by devemux86** [comment]
For the `vtm-ios-example` module we can use a new PR.

**Comment by Longri** [comment]
Sorry, but my example base on Cluster! So I will wait for this!

**Pull request closed without merging**